wordpress-mobile/WordPress-iOS
# Discover posts sorting control

Ref [#14765](https://github.com/wordpress-mobile/WordPress-iOS/issues/14765)
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/340

Basic setup for sorting option with showing the sorting option control in the Reader tab (Discover section) and passing the sorting option when fetching the cards (code added in WPKit PR https://github.com/wordpress-mobile/WordPressKit-iOS/pull/340)

### To test:

Open the Reader tab and check if the control is showing properly.

Things left to check/do:
- [ ] if used strings are localized
- [ ] use proper icons
- [ ] how should the control behave when touched/highlighted
- [ ] set accessibility correctly
- [ ] check if the styling is correct

![Simulator Screen Shot - iPhone 12 Pro - 2021-01-31 at 22 48 55](https://user-images.githubusercontent.com/6242034/106938480-56a17100-671f-11eb-858d-9f64da38d9d3.png)

PR submission checklist:
